x-pack/plugins/cases/public/components/case_view/case_view_tabs.test.tsx

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import type { AppMockRenderer } from '../../common/mock';
+import { createAppMockRenderer } from '../../common/mock';
+import '../../common/mock/match_media';
+import { useCaseViewNavigation } from '../../common/navigation/hooks';
+import type { UseGetCase } from '../../containers/use_get_case';
+import { useGetCase } from '../../containers/use_get_case';
+import { CaseViewTabs } from './case_view_tabs';
+import { caseData, defaultGetCase } from './mocks';
+import type { CaseViewTabsProps } from './case_view_tabs';
+import { userProfiles } from '../../containers/user_profiles/api.mock';
+import { licensingMock } from '@kbn/licensing-plugin/public/mocks';
+import { CASE_VIEW_PAGE_TABS } from '../../../common/types';
+
+jest.mock('../../containers/use_get_case');
+jest.mock('../../common/navigation/hooks');
+jest.mock('../../common/hooks');
+jest.mock('../connectors/resilient/api');
+
+const useFetchCaseMock = useGetCase as jest.Mock;
+const useCaseViewNavigationMock = useCaseViewNavigation as jest.Mock;
+
+const mockGetCase = (props: Partial<UseGetCase> = {}) => {
+  const data = {
+    ...defaultGetCase.data,
+    ...props.data,
+  };
+  useFetchCaseMock.mockReturnValue({
+    ...defaultGetCase,
+    ...props,
+    data,
+  });
+};
+
+export const caseProps: CaseViewTabsProps = {
+  caseData,
+  activeTab: CASE_VIEW_PAGE_TABS.ACTIVITY,
+};
+
+describe('CaseViewTabs', () => {
+  let appMockRenderer: AppMockRenderer;
+
+  beforeEach(() => {
+    mockGetCase();
+
+    const license = licensingMock.createLicense({
+      license: { type: 'platinum' },
+    });
+
+    appMockRenderer = createAppMockRenderer({ license });
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should render CaseViewTabs', async () => {
+    const damagedRaccoonUser = userProfiles[0].user;
+    const caseDataWithDamagedRaccoon = {
+      ...caseData,
+      createdBy: {
+        profileUid: userProfiles[0].uid,
+        username: damagedRaccoonUser.username,
+        fullName: damagedRaccoonUser.full_name,
+        email: damagedRaccoonUser.email,
+      },
+    };
+
+    const license = licensingMock.createLicense({
+      license: { type: 'platinum' },
+    });
+
+    const props = { activeTab: CASE_VIEW_PAGE_TABS.ACTIVITY, caseData: caseDataWithDamagedRaccoon };
+    appMockRenderer = createAppMockRenderer({ features: { metrics: ['alerts.count'] }, license });
```

Comment:
I think `createAppMockRenderer()` will work, right? The component does not use any of the information.